### ISSUE TITLE
Login loop after installing audio script

### ISSUE THREAD

**Issue by murkomarko** [eupnea-linux/audio-scripts#51]
I installed Ubuntu 22.10 with chromeos kernel using depthboot and after getting it all setup I tried fixing sound and now I’m stuck out getting login loop when trying to login with the default Ubuntu gnome interface (wayland), what happens is I type my password, press enter and I get a black screen that shows my password on right top corner and it goes back to login. I can login using Xorg though.

**Comment by murkomarko** [comment]
> 

It’s a Samsung Chromebook plus V2, so according to https://www.chromium.org/chromium-os/developer-information-for-chrome-os-devices/
It’s board name would be Nautilus and it’s baseboard would be poppy

**Comment by WeirdTreeThing** [comment]
Try using the mainline kernel instead.

**Comment by WeirdTreeThing** [comment]
Install `eupnea-mainline-kernel` with your package manager. The audio script in the depthboot image and this script are the exact same.

**Comment by murkomarko** [comment]
I ended up uninstalling and reinstalling a lot of packages related to wayland and eventually I got it working again. One thing I noticed is that the chrome kernel makes the power management work better, for example it suspends the system when the lid is shut and it actually turns off the system when the system is told to. On mainline kernel the computer remains on and has some glitch when the lid is open again, leaving the screen off so I have to force a restart, and when telling the system to shut down it gets stuck on the warning saying the system will shut down, but it never does. Just some two cents for this great project 😊
